index.bs

```diff
@@ -1294,44 +1294,101 @@ A {{ReceiveStream}} has the following internal slots.
    <td class="non-normative">An [=incoming unidirectional=] or [=bidirectional=]
    [=WebTransport stream=].
   </tr>
+  <tr>
+   <td><dfn>\[[Transport]]</dfn>
+   <td class="non-normative">The {{WebTransport}} object owning this {{ReceiveStream}}.
+  </tr>
 </table>
 
 ## Procedures ##  {#receive-stream-procedures}
 
-<div algorithm="create a ReceiveStream">
+<div algorithm>
 
 To <dfn export for="ReceiveStream" lt="create|creating">create</dfn> a
 {{ReceiveStream}}, with an [=incoming unidirectional=] or [=bidirectional=] [=WebTransport stream=]
 |internalStream| and a {{WebTransport}} |transport|, run these steps:
 
 1. Let |stream| be a [=new=] {{ReceiveStream}}, with:
-    : [=[[InternalStream]]=]
+    : [=ReceiveStream/[[InternalStream]]=]
     :: |internalStream|
-1. [=ReadableStream/Set up=] |stream| with TBD.
+    : [=ReceiveStream/[[Transport]]=]
+    :: |transport|
+1. Let |pullAlgorithm| be an action that [=pulls bytes=] from |stream|.
+1. Let |cancelAlgorithm| be an action that [=ReceiveStream/cancels=] |stream|.
+1. [=ReadableStream/Set up with byte reading support=] |stream| with
+   [=ReadableStream/set up with byte reading support/pullAlgorithm=] set to |pullAlgorithm| and
+   [=ReadableStream/set up with byte reading support/cancelAlgorithm=] set to |cancelAlgorithm|.
+1. Set |stream|'s [=ReadableByteStream/[[controller]]=]'s
+   [=ReadableStreamController/autoAllocateChunkSize=] to an [=implementation-defined=] integer.
```

Comment:
Our current plan is for spec-created streams to not use autoAllocateChunkSize, so this line should not be needed.

Reply:
done.